app/controllers/burrows_controller.rb

```diff
@@ -1,10 +1,12 @@
 class BurrowsController < ApplicationController
   def create 
+    
```

Comment:
It's not necessarily wrong but I don't see any reason to use instance variables in your methods.